README.md

```diff
@@ -287,53 +287,161 @@ Although this example is pretty simple and not even includes Lens composition, i
 
 Just as with React, but assuming Angular instead :)
 
+The code for below example - can be found here <https://github.com/georgebatalinski/telescope_todo>
+
 As a bootstrap for the application we need something like:
 
+```typescript 
+@NgModule({
+  declarations: [
+    AppComponent,
+    TodoListComponent
+  ],
+  imports: [
+    ...
+  ],
+  providers: [
+    { provide: 'StreamContract', useFactory: ()=> Telescope.of( AppState.empty() ), deps: [] }
+  ],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+
+We define our global STATE (however, the more ultimate approach would be to have local state)
```

Comment:
I'm not sure what do you mean with "the more ultimate approach". Consider rewording it as a statement of what the example is showing instead of suggesting what users should do.  Also consider renaming STATE to State or IState. All uppercase is sometimes used for constants and it can be confusing to see it as a Type.